### PR TITLE
{numlib}[foss/2019b] Trilinos v12.12.1 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

Requires https://github.com/easybuilders/easybuild-easyblocks/pull/2071

### CODE REVIEW

File: easybuild/easyconfigs/m/MATIO/MATIO-1.5.17-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'MATIO'
+version = '1.5.17'
+
+homepage = 'http://sourceforge.net/projects/matio/'
```
Comment (Micket) [inline easybuild/easyconfigs/m/MATIO/MATIO-1.5.17-GCCcore-8.3.0.eb:14]:
```suggestion
homepage = 'https://sourceforge.net/projects/matio/'
```

Comment (akesandgren) [reply easybuild/easyconfigs/m/MATIO/MATIO-1.5.17-GCCcore-8.3.0.eb]: Done

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 4 out of 6 (4 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/7929f52dd9686cb58f20b45514b941ee for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 2 out of 4 (4 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/e72b5ee1de7b2e1edf6aaab90d49792f for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 2 out of 4 (4 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/600a4ae6c324705e71c6191b1fb58735 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/5ccf281df4eac0ca465c46585def411b for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.8.2003, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/30cb219d7dee8faec646653ade4bb739 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @akesandgren!

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.8.2003, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/eb6885387e26f846b93652ce1b429c2f for a full test report.

**Pull request merged**